aws-lc-sys/builder/cmake_builder.rs

```diff
@@ -135,6 +135,15 @@ impl CmakeBuilder {
 
             cmake_cfg.define("ASAN", "1");
         }
+        match requested_c_std() {
+            CStdRequested::C99 => {
+                cmake_cfg.define("CMAKE_C_STANDARD", "99");
+            }
+            CStdRequested::C11 => {
+                cmake_cfg.define("CMAKE_C_STANDARD", "11");
+            }
+            CStdRequested::None => {}
+        }
```

Comment:
How does the aws-lc-rs' CMake build work? Does it run aws-lc's regular CMakelists.txt? If so why not let it figure out the C standard to use?

Reply:
In most build environments, we will not be setting `CMAKE_C_STANDARD`. This logic would only affect the build when `AWS_LC_SYS_C_STD` has been set to "99" or "11" in the build environment, which should rarely be needed.